README.md

```diff
@@ -83,32 +76,12 @@ Add the following (multi-)task to your `Gruntfile`:
       }
     }
 
-<<<<<<< HEAD
```

Comment:
Great - you cleaned up the merge problems.  Thank you.

Reply:
Oops, sorry, I must've missed it. Git chould detect it itself...